packages/components/src/templates/table.stories.ts

```diff
@@ -0,0 +1,770 @@
+import '../solid-components';
+import { html } from 'lit-html';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+/**
+ * The `sd-table-cell` component offers basic styling for table cells.
+ * It is designed to be used in conjunction with the `sd-table` component.
```

Comment:
Both lines can be removed imo. It's already present in the component documentation

Reply:
Updated